litespi/spi_nor_flash_module.py

```diff
@@ -126,8 +126,17 @@ class SpiNorFlashModule(metaclass=MetaSizes):
 
     Parameters
     ----------
-    spi_nor_cfg : SpiNorConfig
-        User defined configuration for a chip.
+    read_cmd : SpiNorFlashOpCode
```

Comment:
I think you want a `default_read_cmd` and a list for `supported_read_cmds`?

Reply:
I added:
- `default_read_cmd` which is used during init
- `read_cmds` list of commands that can be used by a chip